### DISCUSSION TITLE
使用contents/achievements.tex的问题

### DISCUSSION THREAD

**Discussion by qiaoxiaoyue** [sjtug/SJTUThesis#797] [提问] [answered]
想要把期刊论文和会议论文分开写，所以使用了contents/achievements.tex

> https://github.com/sjtug/SJTUThesis/blob/b874deb48560c3f784d84b54f0786dee372f4ab0/contents/achievements.tex

编译时显示Environment bibliolist undefined. 请问要怎么定义bibliolist？

**Comment by LogCreative** [comment, marked as answer]
考虑使用最新版模板

**Reply by qiaoxiaoyue** [reply]
已经用老模板写完了论文，不是很想换模板了，有什么别的办法嘛？

**Reply by skyzh** [reply]
建议升级一下，理论上都是兼容的。

**Reply by qiaoxiaoyue** [reply]
升级模版嘛？

**Reply by LogCreative** [reply]
替换一下 `texmf` 文件夹的内容

**Reply by qiaoxiaoyue** [reply]
替换后解决，万分感谢
